Ports/julius/package.sh

```diff
@@ -13,11 +13,28 @@ depends=(
 configopts=(
     "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
 )
+data_dir='/home/anon/Games/julius'
+launcher_name='Julius'
+launcher_category='Games'
+launcher_workdir="${data_dir}/"
```

Comment:
I think this is fine if we add a FIXME and a GitHub issue to add a feature to the Launcher service to interpret `$HOME` or similar. or `XDG_DATA_DIR` if we want to be good bois.

Reply:
I'm not sure how this would work in this case because `data_dir` is where we install game data files that should be accessible to everyone, as well as where the game puts user specific data (save files in this case).

Unless I'm misunderstanding what you mean?

Reply:
Ah, I meant for avoiding hard coding /home/anon, we could put a placeholder in the launcher af file. But you're right, that doesn't solve the actual problem of a world readable/writeable location for data files.